src/main/kotlin/com/example/noswimmingserver/domain/teacher/presentation/TeacherController.kt

```diff
@@ -5,10 +5,7 @@ import com.example.noswimmingserver.domain.teacher.presentation.dto.request.Teac
 import com.example.noswimmingserver.domain.teacher.presentation.dto.request.UpdateTeacherInfoRequest
 import com.example.noswimmingserver.domain.teacher.presentation.dto.response.QueryTeacherList
 import com.example.noswimmingserver.domain.teacher.presentation.dto.response.TeacherAccountResponse
-import com.example.noswimmingserver.domain.teacher.service.CreateRandomAccountService
-import com.example.noswimmingserver.domain.teacher.service.QueryTeacherListService
-import com.example.noswimmingserver.domain.teacher.service.TeacherSignInService
-import com.example.noswimmingserver.domain.teacher.service.UpdateTeacherInfoService
+import com.example.noswimmingserver.domain.teacher.service.*
```

Comment:
🚫 **[ktlint]** <no-wildcard-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wildcard import (cannot be auto-corrected)